javascript-review.js

```diff
@@ -4,36 +4,97 @@
 
 // Create an object that has four properties. One property should be set so that the value is a number, another property set to a string, the third to an array, and the fourth to a function. This function should simply console.log the value of the first property.
 
+objectOne = {
+  number: 1,
+  string: "string",
+  array: ["value1", "value2"],
+  function: consoleLogFirstProperty = () => {
+    console.log(objectOne.number)
+  }
+};
+consoleLogFirstProperty();
```

Comment:
careful with your syntax for the function here, see this answer, no need for `consoleLogFirstProperty`

```js
const myObj = {
  number: 1,
  string: "harrison",
  array: [1, 2, 3],
  func: function() {
    console.log(this.number)
  }
}
```